From: Dev Ops <root@localhost>

diff -Naur systemd-239.orig/src/core/shutdown.c systemd-239/src/core/shutdown.c
--- systemd-239.orig/src/core/shutdown.c	2020-06-03 16:19:38.000000000 +0000
+++ systemd-239/src/core/shutdown.c	2020-06-03 16:17:38.000000000 +0000
@@ -323,7 +323,7 @@
 
         log_info("Sending SIGTERM to remaining processes...");
         broadcast_signal(SIGTERM, true, true, arg_timeout);
-
+        sleep(1);
         log_info("Sending SIGKILL to remaining processes...");
         broadcast_signal(SIGKILL, true, false, arg_timeout);
 
